### ISSUE TITLE
Qute error reporting is too verbose

### ISSUE THREAD

**Issue by FroMage** [quarkusio/quarkus#31325]
### Describe the bug

I have 23 errors, and this is the log:

https://gist.github.com/FroMage/efe4825da693adcfa783164fac608dd7

It looks like it's displaying the list of errors three times, and it's also displaying each error twice due to them being stuffed into `Suppressed`.

Can we tweak the error reporting so that we only report the 23 errors once and no exception?

**Comment by FroMage** [comment]
> AFAIK that's the standard way how Quarkus handles suppressed exceptions. They're not useful in the console but since every TemplateException holds some info about the "origin", i.e. line, template, ..., it could be useful for other tools.

Well, can't we find a way to not report them in this case? It makes it really hard to see the nice errors already reported.

**Comment by mkouba** [comment]
> > AFAIK that's the standard way how Quarkus handles suppressed exceptions. They're not useful in the console but since every TemplateException holds some info about the "origin", i.e. line, template, ..., it could be useful for other tools.
> 
> Well, can't we find a way to not report them in this case? It makes it really hard to see the nice errors already reported.

I have no idea. In any case, you can see the error message mentioned in the previous comment at the top of the stack and just ignore all the junk below.

**Comment by FroMage** [comment]
> I have no idea. In any case, you can see the error message mentioned in the previous comment at the top of the stack and just ignore all the junk below.

No, this is terrible UX. We can't do that to our users, we have to do better.

**Comment by mkouba** [comment]
> > I have no idea. In any case, you can see the error message mentioned in the previous comment at the top of the stack and just ignore all the junk below.
> 
> No, this is terrible UX. We can't do that to our users, we have to do better.

Then we should just give up on stack traces in general because it's not a very good UX ;-).

In any case, feel free to propose a better solution.

**Comment by FroMage** [comment]
> Then we should just give up on stack traces in general because it's not a very good UX ;-).

Well, these are different, because they give excellent error messages, then produce a shit load of suppressed exceptions that hide them.

> In any case, feel free to propose a better solution.

We should have a marker interface or annotation on exceptions such as the Qute one, so that the code responsible for printing the exception does not print the stack trace or its suppressed exceptions. The error message is what we want. The rest is gargabe.

**Comment by gastaldi** [comment]
Closing as the current solution seems okay for now.

**Comment by FroMage** [comment]
OK for whom?

**Comment by gastaldi** [comment]
> OK for whom?

I thought it is covered by the changes introduced in https://github.com/quarkusio/quarkus/pull/25810? If I misunderstood please reopen this issue then

**Comment by FroMage** [comment]
No, these changes were made before I reported this issue.
